### PR TITLE
New Resource `alicloud_ecs_elasticity_assurance`; New Datasource `alicloud_ecs_elasticity_assurances`;

### PR DESCRIPTION
**Pull request by super-eggs**
New Resource `alicloud_ecs_elasticity_assurance`; 
New Datasource `alicloud_ecs_elasticity_assurances`;

### CODE REVIEW

File: alicloud/resource_alicloud_ecs_elasticity_assurance.go [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+	if v, err := jsonpath.Get("$.PrivatePoolOptionsId", response); err != nil || v == nil {
+		return WrapErrorf(err, IdMsg, "alicloud_ecs_elasticity_assurance")
+	} else {
+		d.SetId(fmt.Sprint(v))
+	}
+	stateConf := BuildStateConf([]string{}, []string{"Active", "Prepared"}, d.Timeout(schema.TimeoutCreate), 5*time.Second, ecsService.EcsElasticityAssuranceStateRefreshFunc(d, []string{}))
+	if _, err := stateConf.WaitForState(); err != nil {
+		return WrapErrorf(err, IdMsg, d.Id())
+	}
+	return resourceAlicloudEcsElasticityAssuranceRead(d, meta)
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_ecs_elasticity_assurance.go:216]: 看起来tags 不在 create 中啊

File: alicloud/resource_alicloud_ecs_elasticity_assurance_test.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"instance_amount":                     "1",
+					"description":                         "before",
+					"zone_ids":                            []string{"${data.alicloud_zones.default.zones[0].id}"},
+					"private_pool_options_name":           "test_before",
+					"period":                              "1",
+					"private_pool_options_match_criteria": "Open",
+					"instance_type":                       []string{"ecs.c6.large"},
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_ecs_elasticity_assurance_test.go:43]: 用datasource instance_types

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: alicloud/resource_alicloud_ecs_elasticity_assurance_test.go (hunk 2 above)
Comment (super-eggs) [reply alicloud/resource_alicloud_ecs_elasticity_assurance_test.go]: done

File: alicloud/resource_alicloud_ecs_elasticity_assurance.go (hunk 1 above)
Comment (super-eggs) [reply alicloud/resource_alicloud_ecs_elasticity_assurance.go:216]: 有的，在第172行

**Pull request merged**